src/config.ts

```diff
@@ -313,3 +307,70 @@ export const SOFTWARE_VERSION = (() => {
     // into <pre> or <code> where it will create an unnecessary newline.
     return /^(.*)$/m.exec(versionFile)?.at(0) ?? defaultText;
 })();
+
+
+// Command line related functions
+
+/**
+ * Grabs an option from the command line and checks if it exists.
+ * @param arg Option name
+ * @returns True if the option is present, otherwise false.
+ */
+function isCommandLineOptionPresent(args: string[], arg: string): boolean {
+    return args.includes(arg);
+}
+
+/**
+ * Grabs an option's value from program options if it exists, otherwise returns undefined.
+ * @param arg Option name
+ * @param throwOnInvalid If true or undefined, throws an error if the option is present but has no value. If false, returns undefined.
+ * @returns The value passed to the option, or undefined if the option is not specified.
+ * @throws Error if the option is present but has no value.
+ */
+function getCommandLineStringOption(args: string[], arg: string, throwOnInvalid?: boolean): string | undefined {
```

Comment:
We should remove `throwOnInvalid` entirely for this function, if they want to check for a flag, they can just use `isCommandLineOptionPresent` instead.

Reply:
It's there to catch passing no parameter, and consistency

Reply:
It's not a good thing to be consistently confusing.